search/search_repository_whitebox_test.go

```diff
@@ -274,22 +260,24 @@ func (s *searchRepositoryWhiteboxTest) TestSearchByID() {
 	params := url.Values{}
 	ctx := goa.NewContext(context.Background(), nil, req, params)
 	// create 2 work items, the second one having the number of the first one in its title
-	fxt, err := tf.NewFixture(s.DB, tf.WorkItems(2, func(fxt *tf.TestFixture, idx int) error {
-		fxt.WorkItems[idx].Type = workitem.SystemBug
-		fxt.WorkItems[idx].Fields = map[string]interface{}{
-			workitem.SystemTitle:       "Search Test Sbose",
-			workitem.SystemDescription: rendering.NewMarkupContentFromLegacy("Description"),
-			workitem.SystemCreator:     s.modifierID.String(),
-			workitem.SystemAssignees:   []string{"pranav"},
-			workitem.SystemState:       "closed",
-		}
-		fxt.WorkItems[idx].SpaceID = fxt.Spaces[0].ID
-		// for the second work item, use the number of the first work item
-		if idx == 1 {
-			fxt.WorkItems[idx].Fields[workitem.SystemTitle] = "Search Test Sbose" + strconv.Itoa(fxt.WorkItems[0].Number)
-		}
-		return nil
-	}))
+	fxt, err := tf.NewFixture(s.DB,
+		tf.Identities(2, tf.SetIdentityUsernamesFromString([]string{"creator", "assignee"})),
+		tf.WorkItems(2, func(fxt *tf.TestFixture, idx int) error {
+			fxt.WorkItems[idx].Type = workitem.SystemBug
+			fxt.WorkItems[idx].Fields = map[string]interface{}{
+				workitem.SystemTitle:       "Search Test creator",
+				workitem.SystemDescription: rendering.NewMarkupContentFromLegacy("Description"),
+				workitem.SystemCreator:     fxt.Identities[0].ID.String(),
+				workitem.SystemAssignees:   []string{fxt.Identities[1].ID.String()},
```

Comment:
You should use `tf.IdentityByUsername("creator").ID.String()` instead.

Reply:
done